Src/WitsmlExplorer.Frontend/components/Sidebar/SearchFilter.tsx

```diff
@@ -72,6 +72,7 @@ const SeachLayout = styled.div`
   grid-template-columns: 1fr 44px;
   padding: 0.6rem 0rem 0.5rem 1rem;
   position: relative;
+  padding-right: 6px;
```

Comment:
padding for all directions is already defined on line 73 so adding another line for just padding-right is superfluous